server/schema/comment/comment.query.js

```diff
@@ -0,0 +1,76 @@
+const {
+  GraphQLObjectType,
+  // GraphQLScalarType,
+  // GraphQLUnionType,
+  // GraphQLInputObjectType,
+  // GraphQLEnumType,
+  // GraphQLInterfaceType,
+  // GraphQLSchema,
+  GraphQLNonNull,
+  // GraphQLError,
+  GraphQLList,
+  GraphQLString,
+  GraphQLID,
+  // GraphQLBoolean,
+  GraphQLInt,
+  // GraphQLFloat,
+  // GraphQLDate,
+  // GraphQLTime,
+  // GraphQLDateTime,
+  // GraphQLJSON,
+  // GraphQLJSONObject,
+} = require('../scalars');
+const { getListOfComments, getCommentById, countOfComments } = require('./comment.resolver');
+
+const CommentType = require('./comment.type');
+
+module.exports = new GraphQLObjectType({
+  name: 'CommentQuery',
+  fields: {
+    getListOfComments: {
+      description: 'Retrieves comments for given list of ids (default all) in descending order of creation time',
+      type: new GraphQLList(new GraphQLNonNull(CommentType)),
+      args: {
+        ids: {
+          description: 'List of Ids of comments to be retrieved',
+          type: new GraphQLList(new GraphQLNonNull(GraphQLID)),
```

Comment:
@Shurtu-gal GraphQLNonNull should be avoided here the query work fine even if we don't pass the list of ids.